### PR TITLE
Slides for NEUP project meeting on 02-03-21

### PR DESCRIPTION
**Pull request by robfairh**
This is the first draft of a set of slides for today's meeting.

Feel free to commit or suggest any changes here and I will compile the .pdf again.

Here is the [current presentation](https://github.com/arfc/pride/files/5920590/pres.pdf).

### CODE REVIEW

**Review by samgdotson — APPROVED** [review]
These slides look great, thanks for making them @robfairh!!

File: publications/presentations/neup-meeting-02-03-21/slides.tex [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		\begin{center}
+			\includegraphics[height=5.6cm]{images/scenario11.png}
+		\end{center}
+	\end{figure}
+
+\end{frame}
+
+
+\begin{frame}
+\frametitle{Examples of results}
```
Comment (yardasol) [inline publications/presentations/neup-meeting-02-03-21/slides.tex:30]: Maybe add a blurb about how these results are useful to the project?

File: publications/presentations/neup-meeting-02-03-21/slides.tex [hunk 2]
```diff
@@ -0,0 +1,74 @@
+
+\begin{frame}
+\frametitle{Temoa}
+	\begin{itemize}
```
Comment (yardasol) [inline publications/presentations/neup-meeting-02-03-21/slides.tex:4]: Introduce TEMOA a bit (just to frame the presentation)

File: publications/presentations/neup-meeting-02-03-21/slides.tex [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		\item Minimizes the system cost
```
Comment (yardasol) [inline publications/presentations/neup-meeting-02-03-21/slides.tex:5]: Answer the question: what are we using Temoa for?

Comment (yardasol) [reply publications/presentations/neup-meeting-02-03-21/slides.tex:5]: We have a bit of this here already, just need to iron it out an rearrange some bits.

File: publications/presentations/neup-meeting-02-03-21/slides.tex [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+		\item Meets demand of the specified commodities at each time step (electricity, steam, gasoline, etc)
+		\item Meets carbon limits at each time step (optional)
+		\begin{itemize}
+			\item Good at modeling long scenarios (yearly resolution)
+			\item Cannot model transients
+		\end{itemize}
+	\end{itemize}
```
Comment (samgdotson) [inline publications/presentations/neup-meeting-02-03-21/slides.tex]:
```suggestion
			\item Cannot model transients
		\end{itemize}
		\item Temoa is fast, many iterations are possible.
	\end{itemize}
```

File: publications/presentations/neup-meeting-02-03-21/slides.tex [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+				\item with and without alternative vehicles - i.e EVs and FCEVs
+			\end{itemize}
+	\end{itemize}	
+
+    The bigger picture project:
+	\begin{itemize}
+		\item Understand Modelica capabilities for modeling the grid
+		\item Design integration tool to connect Temoa's results with model in Modelica
+		\begin{itemize}
+			\item Modelica has Python capabilities (OMPython and PySimulator Package) that could simplify this integration
```
Comment (yardasol) [inline publications/presentations/neup-meeting-02-03-21/slides.tex:72]:
```suggestion
			\item Modelica has Python capabilities (OMPython and PySimulator Package) that could simplify this integration
			\item How TEMOA fits in this? We can use the longer timesteps of TEMOA to identify priority technology components to model in Modelica based on the output of the TEMOA model.
```

**Comment by robfairh** [comment]
The last version of the slides can be found [here](https://github.com/arfc/pride/files/5920867/pres.pdf).

**Comment by robfairh** [comment]
What should we do about this PR? Should we close it? Should I push it to another branch of ```arfc``` instead? I think it is helpful to  have this template for future meetings.

**Comment by robfairh** [comment]
We had a conversation with @samgdotson IRL concluding that these presentation slides should not be hosted by this repo.

**Pull request closed without merging**